### PR TITLE
Implement further objFuncs

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

File: src/OpenLoco/Objects/TrackObject.h [hunk 1]
```diff
@@ -43,12 +44,13 @@ namespace OpenLoco
         uint8_t num_mods;       // 0x08
         uint8_t num_signals;    // 0x09
         uint8_t mods[4];        // 0x0A
-        uint8_t var_0E;
-        uint8_t pad_0F[0x14 - 0x0F];
-        uint16_t build_cost_factor;  // 0x14
-        uint16_t sell_cost_factor;   // 0x16
-        uint16_t tunnel_cost_factor; // 0x18
-        uint8_t cost_index;          // 0x1A
+        uint16_t var_0E;        // ?compatible signals bitset?
```
Comment (duncanspumpkin) [inline src/OpenLoco/Objects/TrackObject.h:47]: possible source of a bug here as we were accessing this incorrectly as uint8_t

**Comment by LeftofZen** [comment]
Testing this branch, I get an interop crash on game startup, trying to narrow down the source now

File: src/OpenLoco/Objects/TrackObject.cpp [hunk 2]
```diff
@@ -13,4 +14,64 @@ namespace OpenLoco
...
+    }
+
+    // 0x004A6A5F
+    void TrackObject::load(const LoadedObjectHandle& handle, stdx::span<std::byte> data)
+    {
+        Interop::registers regs;
+        regs.esi = Interop::X86Pointer(this);
+        regs.ebx = handle.id;
+        regs.ecx = enumValue(handle.type);
+        Interop::call(0x0042F4D0, regs);
```
Comment (duncanspumpkin) [inline src/OpenLoco/Objects/TrackObject.cpp:62]: Missmatch between function call and offset address...

**Review by LeftofZen — CHANGES_REQUESTED** [review]
Merge conflicts

**Review by LeftofZen — APPROVED** [review]
tested on a few maps/games, couldn't find any issues with it

**Pull request merged**